app/controllers/concerns/attachments_copier.rb

```diff
@@ -0,0 +1,19 @@
+module AttachmentsCopier
+  def copy_attachments(record)
+    record.content.scan(Attachment::SCREENSHOT_REGEX).each do |screenshot_url|
+      _, _, _, _, project_id, node_id, filename, _ = screenshot_url
```

Comment:
```suggestion
      screenshot_path, _, _, _, project_id, node_id, filename, _ = screenshot_url
```

Reply:
We can name this part so it can be used on L10 and L15. screenshot_path is a suggestion but depends what you decide to go with on L3